app/views/passwords/edit.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1>Reset Your Password</h1>
+<div class="w-25 container">
+  <%= form_with model: @user, url: password_edit_path(token: params[:token]) do |form| %>
+    <%if @user.errors.any? %>
+      <%= render 'layouts/flash', message: flash%>
+    <% end %>
+    <%= form.label :password  %><br>
+    <div class="mb-3">
+      <%= form.password_field :password, class: "form-control" %>
+    </div>
+    <%= form.label :password_confirmation %><br>
+    <div class="mb-3">
+      <%= form.password_field :password_confirmation, class: "form-control" %>
+    </div>
+    <div>
+    <%= form.submit "Reset Password" %>
+    </div>
+    <%= render 'layouts/flash', message: flash%>
```

Comment:
Move this logic to layout file so that you don't need to render it in each view file. You can render the flash messages on top.